malib/manager/rollout_worker_manager.py

```diff
@@ -70,12 +70,30 @@ def __init__(
                 worker_index=worker_idx,
                 env_desc=self._env_desc,
                 metric_type=self._metric_type,
+                test=False,
```

Comment:
what's the functionality of `test`?

Reply:
It is used for creating rollout workers for deterministic evaluation.
Please also refer to:

Reply:
It is used for creating rollout workers for deterministic evaluation.
Please also take a look at:
https://github.com/apexrl/malib/blob/196de6592fd82ea889cb871d4663d9e4d5028dde/malib/rollout/base_worker.py#L339